### PR TITLE
Update evaluation-to-prod.adoc

### PR DESCRIPTION
**Pull request by xaviergully**
To be consistent with the rest of the documentation and the different chapters

### CODE REVIEW

File: docs/ota-client-guide/modules/ROOT/pages/evaluation-to-prod.adoc [hunk 1]
```diff
@@ -20,15 +20,15 @@ Here are our recommendations for xref:intro-evaluate.adoc[evaluating OTA connect
...
+== Build your own OTA-enabled solution
 
 === Recommendations
 
 Here are our recommendations for xref:intro-prep.adoc[integrating OTA Connect into your production environment]:
 
 include::partial$recommended-steps.adoc[tags=integrate-steps]
 
-== Deploy
+== Deploy your own OTA-enabled solution
```
Comment (merltron) [inline docs/ota-client-guide/modules/ROOT/pages/evaluation-to-prod.adoc]:
```suggestion
== Deploy your OTA-enabled solution
```
You don't need to say "your own" twice. It's just for the the "build" chapter.

**Comment by patrickvacek** [comment]
Missing the DCO. @xaviergully can you add that if possible?

**Comment by xaviergully** [comment]
I have been trying but did not succeed.
I think I correctly set it up on my machine but does not seem to work

**Comment by patrickvacek** [comment]
> I have been trying but did not succeed.
> I think I correctly set it up on my machine but does not seem to work

Interesting, the DCO check is successful now, despite that I don't actually see the signoff. I can take a look at your setup today when you have time if you'd like.

**Comment by xaviergully** [comment]
> > I have been trying but did not succeed.
> > I think I correctly set it up on my machine but does not seem to work
> 
> Interesting, the DCO check is successful now, despite that I don't actually see the signoff. I can take a look at your setup today when you have time if you'd like.

I just pressed on the DCO button to resolved the issue :s

**Pull request merged**